src/include/mempool.h

```diff
@@ -253,6 +255,84 @@ class pool_t {
 void dump(ceph::Formatter *f, size_t skip=2);
 
 
+// Object pool for placing storing instances in continuous memory. The class
```

Comment:
@rzarzynski radoslaw I believe multiline comments should be inside /* */
Though old code is full of // But we can improve while writing new one.